### DISCUSSION TITLE
Failed to decompress

### DISCUSSION THREAD

**Discussion by yao1576** [seuyh/stellaris-dlc-unlocker#39] [Q&A]
After the download is complete, an extraction error is displayed
![Uploading 屏幕截图 2024-12-20 200310.png…]()

**Comment by lsy-404** [comment]
You failed upload picture. Could you try again?
你没上传成功（）

**Reply by yao1576** [reply]
![屏幕截图 2024-12-20 200239](https://github.com/user-attachments/assets/40f2b481-1cd8-48b9-b7db-6f2c22b75786)

**Comment by lsy-404** [comment]
#40 
Transfer to issue
转移到问题
